app/pages/home/landing.jsx

```diff
@@ -41,27 +45,19 @@ const LogoImage = styled.img`
     z-index: 99;
 `;
 
-const Text = styled.h2`
-    fontSize: 50px;
-    color: white;
-    text-align: center;
```

Comment:
If you could make a styled SectionHeader here and add the css text-align: center to make it look like the current master branch.